server_tests/test/clientTests.js

```diff
@@ -1918,7 +1965,12 @@ describe("RUST API TEST SUITE", function() {
             }));
         });
 
-        it('Expect success. camera._bublShutdown successfully returned', function(done) {
+        it('Expect success. camera._bublShutdown successfully returned. Disabled over wifi', function(done) {
+            if (testViaWifi) {
```

Comment:
I'm thinking we need an additional parameter for this. Maybe a model = "bublMock". Whether we're on wifi or not doesn't change the fact that the camera shutting down during the test is a problematic thing.